2.segment_images/scripts/0.segment_nuclei.py

```diff
@@ -127,13 +137,17 @@
 
 # ## Cellpose
 
-# In[ ]:
+# In[6]:
 
 
 use_GPU = torch.cuda.is_available()
 # Load the model
 model_name = "nuclei"
-model = models.CellposeModel(gpu=use_GPU, model_type=model_name)
+
+model = models.CellposeModel(
+    gpu=use_GPU,
+    model_type=model_name,
```

Comment:
If running multiple jobs in a multi GPU system is a thing for this project, making adding a cellpose argument `device = torch.cuda.device(i)` (and make `i` a parse-able notebook argument maybe) would be a good idea,
`torch.cuda.device_count()` can be used cap `i`  

```suggestion
    model_type=model_name,
    device=torch.cuda.device(0) if use_GPU else None
```
In the suggestion all jobs will be run with the default first cuda device